vars/fcosKola.groovy

```diff
@@ -3,6 +3,7 @@
 //    cosaDir:         string  -- cosa working directory
 //    parallel:        integer -- number of tests to run in parallel (default: 8)
 //    skipUpgrade:     boolean -- skip running `cosa kola --upgrades`
+//    selfTests:      []string -- list of test paths to run
```

Comment:
Ahh OK, one thing I didn't realize here was that the self-tests themselves were located in cosa under `tests/kola-ci-self` and not as part of kola itself.

Hmm, in that case, how about we generalize this slightly so that it's just a parameter that takes paths that we pass to the main kola run. E.g. something like

```groovy
// addExtTests []string -- list of dirs containing additional external tests
...
def call(params = [:]) {
    ...
    def addExtTests = params,get('addExtTests', [])
    ...
    for (path in addExtTests) {
        args += "--exttest ${path}"
    }
    ...
}
```

?

Then, CI for cosa can just pass the `tests/kola-ci-self` dir when it runs kola, or if it really wants the run to be separate from the main run, call `fcosKola` again with that and an `extraArgs` set to `ext.kola-ci-self*` so it only runs those tests.

Does that make sense?